site/en/docs/lighthouse/seo/is-crawlable/index.md

```diff
@@ -39,11 +39,12 @@ X-Robots-Tag: noindex
 You might also have `<meta>` elements that block specific crawlers, such as:
 
 ```html
-<meta name="AdsBot-Google" content="noindex"/>
+<meta name="Googlebot" content="noindex"/>
 ```
 
-Lighthouse doesn't check for crawler-specific directives like this, but they can
-still make your page harder to discover, so use them with caution.
+Lighthouse doesn't fail the audit for crawler-specific directives like this, but they can
```

Comment:
Shoudl we qualify this as "as of 10.0"? Must account for chrome stable running older LH longer than other channels; and to a lesser extent, historical LH versions.

Reply:
I don't think we need a 10.0 disclaimer for something as minor as this